training/distributed_training/pytorch/data_parallel/rnnt/RNNT_notebook.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<p>It's cleaner to use hyperlinks rather than raw links - use the doc page as the hyperlink text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>fixed thanks!</p>